### PR TITLE
Added new repository: Okami – Adaptive Color Schemes

### PR DESCRIPTION
**Pull request by Rayraegah**
I've added two sets of two dark and light colour schemes for Sublime Text 3. The colour schemes are written in `.sublime-color-scheme` syntax. You'll find these files in the [schemes](https://github.com/Rayraegah/okami/tree/master/schemes) folder of my github repository [or in the releases](https://github.com/Rayraegah/okami/releases)

A [readme.md](https://github.com/Rayraegah/okami/blob/master/README.md) file is included in the root of the git repository and it contains a brief description, screenshots, and installation instructions.

### CODE REVIEW

**Comment by Rayraegah** [comment]
Moved packages to root directory. Release at v1.0.1

**Comment by Rayraegah** [comment]
Looks like the bot does not recognise the `.sublime-color-scheme` files yet https://github.com/packagecontrol/st_package_reviewer/pull/20 

On that note though could the scheme files exist in their own directory? (not necessarily on the project root)

**Comment by Thom1729** [comment]
> On that note could the scheme files exist in their own directory? (not necessarily on the project root)

Yes, and this is probably better (though certainly not required) for the sake of organization.

**Comment by Rayraegah** [comment]
> Yes, and this is probably better (though certainly not required) for the sake of organization.

Done.

File: repository/o.json [hunk 1]
```diff
@@ -234,7 +241,18 @@
...
+				}
+			]
+		},
+		{
+			"name": "Okami – Adaptive Color Schemes",
+			"details": "https://github.com/Rayraegah/okami",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": ">=3176",
```
Comment (FichteFoll) [inline repository/o.json]: Unfortunately, this excludes everyone on the dev builds despite the latest being practically equivalent with the latest on the stable branch. I'll change it to 3153, where the new format was added.

**Pull request merged**